templates/CRM/Contact/Form/Search/Criteria/Basic.tpl

```diff
@@ -23,6 +23,13 @@
  | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
  +--------------------------------------------------------------------+
 *}
+{literal}
+<style type="text/css">
+  .adv-search-top-submit span.crm-button {
```

Comment:
civicrm.css is over 4000 lines long, which is about 3000 lines longer than any css file has a right to be.
I'm struggling with what to do about that, but it's clear to me that we need to stop adding "single-use" styles into that file. This is a perfect example - this is a style that is only used in 1 single place on 1 single form, so it has no right to be in a global document like civicrm.css which is loaded everywhere (contribution forms, event registration pages, all sorts of unrelated contexts).
Probably in this case the right fix would be to start a `searchForm.css` file and move any existing "single-use" search form styles into it. NB I've already done that for `contactSummary.css` and it resulted in about 400 lines removed from civicrm.css!
But I'm less sure about other places. I'm not really advocating for this, but out of curiosity, since our tpls are peppered with inline `<script>` tags and you don't seem to have a problem with that, what's wrong with also sprinkling in `<style>` tags? What's the difference?